docs/userGuide/plugins/mathDelimiters.md

```diff
@@ -0,0 +1,53 @@
+### Plugin: MathDelimiters 
+
+This plugin allows you to use additional delimiters for your math formulae. Availble delimiters are:
+
+* `'beg_end'`
```

Comment:
This list looks a little a awkward since there are a lot more options in the package.

Should we remove it? 🤔  Your example below seems sufficient